### PR TITLE
changes to the ageing recipes metadata

### PR DESCRIPTION
**Pull request by JimBryant7**
### What

Changed some of the wording around the ageing recipes to match the BA needs. Made some changes to the ageing recipes variables. As these datasets have only gone into develop it won't have a large knock on effect.

Have also added 8 new GDP & GVA recipes. The "sic-test" code list being used is correct

### How to review

Sanity check

### Who can review

Anyone

### CODE REVIEW

File: recipe/data.go [hunk 1]
```diff
@@ -1753,19 +1753,19 @@ var AgeingSingleHouseholds = Response{
 	},
 }
 
-// Local authority ageing statistics - sex ratios for people over 65 and over 85
+// Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over
 var AgeingSexRatios = Response{
 	ID:     "df8c8df5-484a-4d29-b246-a7720caf2474",
-	Alias:  "Local authority ageing statistics - sex ratios for people over 65 and over 85",
+	Alias:  "Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over",
```
Comment (janderson2) [inline recipe/data.go]: Should there be a comma here:
```suggestion
	Alias:  "Local authority ageing statistics, sex ratios for people aged 65 and over, and 85 and over",
```

File: recipe/data.go [hunk 2]
```diff
@@ -1753,19 +1753,19 @@ var AgeingSingleHouseholds = Response{
...
 	InputFiles: []file{
 		{"AgeingSexRatios"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ageing-sex-ratios",
+			DatasetID: "older-people-sex-ratios",
 			Editions:  []string{"time-series"},
-			Title:     "Local authority ageing statistics, sex ratios for people over 65 and over 85",
+			Title:     "Local authority ageing statistics, sex ratios for people aged 65 and over and 85 and over",
```
Comment (janderson2) [inline recipe/data.go]: Should there be a comma here:
```suggestion
			Title:     "Local authority ageing statistics, sex ratios for people aged 65 and over, and 85 and over",
```

File: recipe/data.go [hunk 3]
```diff
@@ -1788,19 +1788,19 @@ var AgeingSexRatios = Response{
 	},
 }
 
-// Local authority ageing statistics - net flows for over 65s and over 85s
-var AgeingNetFlows = Response{
+// Local authority ageing statistics - net internal migration for over 65s and over 85s
+var AgeingNetInternalMigration = Response{
 	ID:     "c5875623-0874-429d-bbc6-63d2d84e6159",
-	Alias:  "Local authority ageing statistics - net flows for over 65s and over 85s",
+	Alias:  "Local authority ageing statistics - net internal migration for over 65s and over 85s",
```
Comment (janderson2) [inline recipe/data.go]: We are using `over 65s and over 85s` here, but we use `65 and over and 85 and over` above.  Potentially worth considering aligning them for consistency.

File: recipe/data.go [hunk 4]
```diff
@@ -1788,19 +1788,19 @@ var AgeingSexRatios = Response{
...
-		{"AgeingNetFlows"},
+		{"AgeingNetInternalMigration"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ageing-net-flows",
+			DatasetID: "older-people-net-internal-migration",
 			Editions:  []string{"time-series"},
-			Title:     "Local authority ageing statistics, net flows for over 65s and over 85s",
+			Title:     "Local authority ageing statistics, net internal migration for over 65s and over 85s",
```
Comment (janderson2) [inline recipe/data.go]: Same comment as above on consistency.

File: recipe/data.go [hunk 5]
```diff
@@ -1713,19 +1713,19 @@ var AgeingPopProj = Response{
...
 	InputFiles: []file{
 		{"AgeingSingleHouseholds"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ageing-single-households",
+			DatasetID: "older-people-in-single-households",
 			Editions:  []string{"time-series"},
-			Title:     "Local authority ageing statistics, percentage of single households headed by someone over 65 and over 85",
+			Title:     "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over",
```
Comment (janderson2) [inline recipe/data.go]: Should we use a comma here:
```suggestion
			Title:     "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over, and 85 and over",
```

File: recipe/data.go [hunk 6]
```diff
@@ -1713,19 +1713,19 @@ var AgeingPopProj = Response{
 	},
 }
 
-// Local authority ageing statistics - % of single households headed by someone over 65 and over 85
+// Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over
 var AgeingSingleHouseholds = Response{
 	ID:     "226d6940-ed0c-4556-91f1-9ca61d56c8a0",
-	Alias:  "Local authority ageing statistics - % of single households headed by someone over 65 and over 85",
+	Alias:  "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over and 85 and over",
```
Comment (janderson2) [inline recipe/data.go]: Should there be a comma here:
```suggestion
	Alias:  "Local authority ageing statistics, percentage of single households headed by someone aged 65 and over, and 85 and over",
```

File: recipe/data.go (hunk 1 above)
Comment (JimBryant7) [reply recipe/data.go]: comma probably makes sense but i have used the wording as the business area wanted it..

File: recipe/data.go (hunk 2 above)
Comment (JimBryant7) [reply recipe/data.go]: same as previous

File: recipe/data.go (hunk 3 above)
Comment (JimBryant7) [reply recipe/data.go]: business area made no request to change this but it does make sense to align them so i have

File: recipe/data.go (hunk 4 above)
Comment (JimBryant7) [reply recipe/data.go]: same as above

File: recipe/data.go (hunk 5 above)
Comment (JimBryant7) [reply recipe/data.go]: same as earlier

File: recipe/data.go (hunk 6 above)
Comment (JimBryant7) [reply recipe/data.go]: same as previous

**Pull request merged**